<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Victor Hunt</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/img/img/logo.png</normaloff>:/img/img/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #ffffff;</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="LabelTicket">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/img/img/logo.png);
background-color: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btCreate">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Black</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: black
;
color: #ffffff;
border-radius: 10px;

</string>
    </property>
    <property name="text">
     <string>start</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: none;
color: #AA0000;</string>
    </property>
    <property name="text">
     <string>Hunt</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="picture_people">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>471</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/img/img/people.png);
height: 229px;
width: 430px;
background-color: none;
  background-position: center; /* Center the image */
  background-repeat: no-repeat; /* Do not repeat the image */
  background-size: cover</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #AA0000;
background-color: none;</string>
    </property>
    <property name="text">
     <string>Victor</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>290</y>
      <width>541</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {
     border: none;
     border-radius: 5px;
     background-color: #ffffff;
 }

 QProgressBar::chunk {
     background-color: #FFC100;
     width: 20px;
 }</string>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>270</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #3A3B3C;</string>
    </property>
    <property name="text">
     <string>1.2.4</string>
    </property>
   </widget>
   <zorder>picture_people</zorder>
   <zorder>LabelTicket</zorder>
   <zorder>label_3</zorder>
   <zorder>label</zorder>
   <zorder>btCreate</zorder>
   <zorder>progressBar</zorder>
   <zorder>label_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>526</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
